stl/src/vector_algorithms.cpp

```diff
@@ -3261,6 +3261,156 @@ namespace {
 
         return _Result;
     }
+
+    template <class _Traits, class _Ty>
+    const void* __stdcall __std_search_impl(

```

Comment:
No change requested. Should we starts to use `pretty` names in the implementation details given these stuffs are in an unnamed namespace in a separately compiled source file?

Reply:
A counter argument could be that we might decide to vectorize in heafers, like libc++ doea. Otherwsie I'm fine with that

Reply:
At a minimum, we should be consistent within a file. Intra-file inconsistency is intolerable.